doc/MANUAL.adoc

```diff
@@ -314,10 +314,28 @@ this:
    `$HOME/.ccache/ccache.conf`.
 5. Otherwise, if `XDG_CONFIG_HOME` is set then use
    `$XDG_CONFIG_HOME/ccache/ccache.conf`.
-6. Otherwise, use `%APPDATA%/ccache/ccache.conf` (Windows),
+6. Otherwise, use
    `$HOME/Library/Preferences/ccache/ccache.conf` (macOS) or
    `$HOME/.config/ccache/ccache.conf` (other systems).
 
+On Windows, this is the method used:
+
+1. If `CCACHE_CONFIGPATH` is set, use that path.
+2. Otherwise, if the environment variable `CCACHE_DIR` is set then use
+   `$CCACHE_DIR/ccache.conf`.
+3. Otherwise, if <<config_cache_dir,*cache_dir*>> is set in the secondary
+   (system-wide) configuration file then use `<cache_dir>\ccache.conf`. The
+   system-wide configuration on Windows is
+   `$env:ALLUSERSPROFILE\ccache\ccache.conf` by default, `$env:ALLUSERSPROFILE`
```

Comment:
If I understand correctly, the `$env:VAR` syntax is what's used in PowerShell and doesn't work in cmd? Do you think more people will understand that syntax than `%VAR%`? (I didn't until I googled.) And if they're using Bash (Git for Windows) it's yet a different syntax... :thinking: Perhaps we can reformulate the documentation to be agnostic to variable derefencing syntax?

Reply:
Fixed.